lib/fxpay/settings.js

```diff
@@ -49,6 +49,8 @@
     },
     // Reference window so tests can swap it out with a stub.
     window: window,
+    winWidth: 276,
```

Comment:
maybe preface this with a comment saying how this is used when there's no native window opener